src/main/java/seedu/address/model/date/Date.java

```diff
@@ -59,8 +59,8 @@ public int compareTo(Date d) {
     }
 
     /**
-     * Returns the String representation of the Date in format suitable for storage logging
-     * @return String
+     * Returns the String representation of the Date in a format suitable for storage logging.
+     * @return String the string that will be logged.
```

Comment:
"will be logged" sounds like this method does the logging. Should rephrase it to

```suggestion
     * @return String the string that is in the correct format for logging.
```